### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by RainRat**

### CODE REVIEW

**Review by melvinebenezer — CHANGES_REQUESTED** [review]

File: safety/README.md [hunk 1]
```diff
@@ -26,14 +26,14 @@ community.
 ## Why create a safety pipeline?
 
 Open source and extendable safety pipelines unfortunately do not exist on the
-same on the same scale as those in ChatGPT and other commerical systems. To
+same on the same scale as those in ChatGPT and other commercial systems. To
```
[the remark this answers is not in the document]
Comment (RainRat) [reply safety/README.md]: yes. added to change.

**Review by andrewm4894 — APPROVED** [review]
Thanks!

**Comment by andrewm4894** [comment]
Might need to try run pre-commit to try get it passing.

**Review by SummerSigh — APPROVED** [review]
Looks good to me!

**Comment by andreaskoepf** [comment]
@RainRat the llama file has been removed, could you please resolve the conflict? thx :-)

**Comment by RainRat** [comment]
Ok, deleted the llama file.

**Review by andreaskoepf — APPROVED** [review]
Thanks a lot!

**Pull request merged**